data/saas/dataset/alchemer_dataset.yml

```diff
@@ -0,0 +1,78 @@
+dataset:
+  - fides_key: <instance_fides_key>
+    name: alchemer_erasure_only
+    description: >-
+      A sample dataset representing the Alchemer erasure only connector for
+      Fides
+    collections:
+      - name: user
```

Comment:
By defining fields in this collection we're saying that this integration will return access data. If that's the case, make sure to include the `read` request in the config, otherwise remove these fields from the collection